sdk/identity/azure-identity-perf/src/main/java/com/azure/identity/perf/core/ServiceTest.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.perf.core;
+
+import com.azure.core.credential.TokenRequestContext;
+import com.azure.identity.InteractiveBrowserCredential;
+import com.azure.identity.InteractiveBrowserCredentialBuilder;
+import com.azure.perf.test.core.PerfStressOptions;
+import com.azure.perf.test.core.PerfStressTest;
+import reactor.core.publisher.Mono;
+
+public abstract class ServiceTest<TOptions extends PerfStressOptions> extends PerfStressTest<TOptions> {
+    protected static final String CLI_CLIENT_ID = "04b07795-8ddb-461a-bbee-02f9e1bf7b46";
```

Comment:
do we need them to be static ?

Reply:
I'm simply defining it as a constant: https://stackoverflow.com/questions/9639007/defining-constant-string-in-java